### DISCUSSION TITLE
`terminal` and tabs

### DISCUSSION THREAD

**Discussion by JeffersGlass** [pyscript/pyscript#1829] [General]
_[Warning] this discussion could be a can of worms🥫🪱. I am opening it as a discussion because I **don't** think it's an error, nor a release blocker, but maybe an improvement._

As of the merge of #1816 (hooray!) we can have a functional REPL, running in a worker, and outputting to a terminal on the screen. This is awesome.

Currently, hitting `tab` has no effect - it doesn't insert a visible tab (or spaces) in the line, nor does it change focus on the page to the next element. 

As a person who opts for tabs over spaces, I would like it to insert either a tab character in the terminal, or perhaps four spaces as a compromise. If it's not going to do either, it might as well change focus on the page.

Thoughts?

**Comment by WebReflection** [comment]
Is this something XTerm related? I honestly have zero hooked in control over special chars and even ctrl+r to reload does nothing because the XTerm seems to take over entirely everything a user types.
